Fix path for iscsid as we ship it in /usr/sbin
Index: open-iscsi/etc/iscsid.conf
===================================================================
--- open-iscsi.orig/etc/iscsid.conf	2013-11-05 20:56:42.941418667 +0530
+++ open-iscsi/etc/iscsid.conf	2013-11-05 20:56:42.941418667 +0530
@@ -20,7 +20,7 @@
 # iscsid.startup = /etc/rc.d/init.d/iscsid force-start
 # 
 # Default for upstream open-iscsi scripts (uncomment to activate).
-iscsid.startup = /sbin/iscsid
+iscsid.startup = /usr/sbin/iscsid
 
 
 #############################
